appengine-java11/README.md

```diff
@@ -0,0 +1,67 @@
+# Google App Engine Standard Environment Samples for Java 11
+
+<a href="https://console.cloud.google.com/cloudshell/open?git_repo=https://github.com/ludoch/samples&page=editor&open_in_editor=appengine-java11/README.md">
+<img alt="Open in Cloud Shell" src ="http://gstatic.com/cloudssh/images/open-btn.png"></a>
+
+This is a repository that contains Java code samples for [Google App Engine][ae-docs]
+standard Java 11 environment.
+
+[ae-docs]: https://cloud.google.com/appengine/docs/java/
+
+## Prerequisites
+
+**Private Alpha**
+To access the Java 11 Google App Engine standard runtime, you'll need to apply
+to become part of the App Engine [Alpha program](https://docs.google.com/forms/d/e/1FAIpQLSf5uE5eknJjFEmcVBI6sMitBU0QQ1LX_J7VrA_OTQabo6EEEw/viewform).
+
+
+### Download Maven
```

Comment:
You might consider also doing Gradle for all of these.

Reply:
On my to do list.